src/macOSNotif.js

```diff
@@ -446,7 +419,7 @@ class macOSNotifJS {
 }
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
As the below is now set as an attribute of window, this eslint line shouldn't be needed?